lib/member_page.rb

```diff
@@ -36,11 +36,15 @@ class MemberPage < Scraped::HTML
   end
 
   field :cell do
-    noko.xpath("//tr/td[contains(., 'Mobile Phone')]").text.match(/Mobile Phone[[:space:]]*?\:[[:space:]]*?([^\nH]+)/).to_a[1].to_s.tidy.gsub(' or', ';')
+    noko.xpath("//tr/td[contains(., 'Mobile Phone')]").text
+        .match(/Mobile Phone[[:space:]]*?\:[[:space:]]*?([^\nH]+)/)
+        .to_a[1].to_s.tidy.gsub(' or', ';')
   end
 
   field :phone do
```

Comment:
this implementation is a bit too complex to not be tested. (Both it and `:cell` would also benefit from a comment explaining why they're so complex — probably just by showing examples of what the different formats are you're matching. That will make it a LOT easier to make sure in future, when we need to adapt it for an example where the number isn't being caught, or if the layout changes.